security-oauth2-quickstart/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
-quarkus.oauth2.client-id=client_id
+quarkus.oauth2.enabled=true
+quarkus.oauth2.client-id=backend-service
 quarkus.oauth2.client-secret=secret
-quarkus.oauth2.introspection-url=http://oauth-server/introspect
\ No newline at end of file
+quarkus.oauth2.introspection-url=http://localhost:8180/auth/realms/quarkus/protocol/openid-connect/token/introspect
```

Comment:
Wondering if this property can use a random port (and set by a Quarkus Test Resource)

Reply:
This configuration is overwritten during at QuarkusTestResource creation. I was not able to delete this configuration because the following error occurs if not written.
```
io.quarkus.runtime.configuration.ConfigurationException: client-id, client-secret and introspection-url must be configured when the oauth2 extension is enabled
```